src/helm/clients/anthropic_client.py

```diff
@@ -327,8 +341,24 @@ def do_it() -> Dict[str, Any]:
                 },
                 request,
             )
-
             response, cached = self.cache.get(cache_key, wrap_request_time(do_it))
+
+            if _is_content_moderation_failure(response):
+                hlog(
+                    f"WARNING: Returning empty request for {request.model_deployment} "
+                    "due to content moderation filter"
+                )
+                return RequestResult(
+                    success=False,
+                    cached=False,
```

Comment:
`cached=cached` rather than `cached=False` since moderation filter responses _can be cached_.